app/views/vendors/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :sidebar do %>
+  <%= sidebar_button_link 'New Vendor', new_vendor_path if can? :create, Vendor %>
+<% end %>
+
+<table id="List_of_vendors" class="generic_datatable table table-striped table-bordered">
+  <caption><h3><%= @page_title %></h3></caption>
+
+  <thead>
+    <tr>
+      <th>List of Vendors</th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+    <% Vendor.all.each do |vendor| %>
```

Comment:
in the controller you assign `@vendors = Vendor.all` but then you don't use `@vendors` anywhere